site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,409 @@
+
+The groundwork for this change had already been put in place with a PR [jroweboy](https://github.com/jroweboy) made for [vsync presentation](https://github.com/citra-emu/citra/pull/4940) back in 2019. The vsync presentation PR was created as a way to force enable vsync for everyone from within Citra. In the past, this was disabled by default. 
+For those who force enabled vsync in their driver settings, Citra wasn’t able to disable vsync, which resulted in inconsistent frame limiting. 
+This was picked up by [epicboy](https://github.com/ameerj) and molded to work with multiple windows being displayed by one instance of Citra.
+
+To enable this feature, head to `View -> Screen Layout -> Separate Windows` from within Citra. Launch your game, and from there you can edit the size and location of the windows to wherever you want on your PC!
+
+{{< figure src="screens.png"
+    title="Map on one screen, gameplay on the other!" >}}
+
+### Ask for confirmation when changing games from the game list ([#6186](https://github.com/citra-emu/citra/pull/6186)) by [foghawk](https://github.com/foghawk)
+
+When you run Citra without Single Window Mode enabled, you have the game window and the game list window open at the same time. If you clicked another game in the game list, it’d instantly boot, closing your current game. 
+
+After experiencing this themselves via accidentally clicking onto another game whilst already in a game, [foghawk](https://github.com/foghawk) added a confirmation check. No longer will you lose any important save data from this issue, and you can carry on playing after selecting one of the two options!
+
+{{< figure src="stopEmu.png"
+    title="No more accidentally losing your progress." >}}
+	
+## Per-game configurations ([#6187](https://github.com/citra-emu/citra/pull/6187), [#6219](https://github.com/citra-emu/citra/pull/6219)) by [GPUCode](https://github.com/GPUCode)
+
+Based off of the yuzu implementation of the same name, Per-Game Configurations have landed in Citra! This is part of a series of changes coming to Citra to modernize the frontend of the emulator.
+
+Tired of having to change your settings for each game you play every time you launch it? Well, worry no more! With this change to Citra, each of your games now has their own settings profile. 
+You can change the emulation speed, change graphical settings, audio settings such as what audio emulation you’re using, the volume of each game, and so many more!
+
+Individual config files for each per-game setting you may have are stored in `config -> custom -> <titleid>.ini` These .ini files can be edited using a text editor application, and the changes will appear in Citra.
+
+This is another addition which has been highly requested in the Citra community, and we are proud that it is finally here.
```

Comment:
Before this PR, users would simulate a speed up button by using a hotkey to switch between Emulation Speed and Alternate Speed with a hotkey (Toggle Alternate Speed). With one of the two settings being set higher than the other.

With this PR, the option for Alternate Speed in the general settings has been removed. We now have a Per-game Emulation Speed option which sorta fulfills the same purpose. 
The "Toggle Alternate Speed" hotkey has also been renamed to `Toggle Per-Game Speed` in https://github.com/citra-emu/citra/pull/6224 (<-- this PR isn't mentioned in the progress report, needs to be added as well).